### PR TITLE
NO_JIRA: Refactored member visible column config.

### PR DESCRIPTION
**Pull request by pepcus-priya**
# Related issues
-  Refactored member visible column config.

# Proposed changes
- Refactored member visible column config.
- Added Member Info Buttons.

### CODE REVIEW

File: src/v2/layouts/MemberColumnConfig/MemberColumnConfig.tsx [hunk 1]
```diff
@@ -0,0 +1,164 @@
+import { Modal } from '../../components/design/modal/Modal';
```
Comment (pepcus-mohit) [inline src/v2/layouts/MemberColumnConfig/MemberColumnConfig.tsx:1]: This directory/file should be inside MemberDataGrid directory.

File: src/v2/layouts/MemberConfigSearch/MemberConfigSearch.tsx [hunk 2]
```diff
@@ -0,0 +1,201 @@
+import { CheckboxLabel } from '../../components/design/form/CustomInputs/FormElements/Checkbox/CheckboxLabel';
```
Comment (pepcus-mohit) [inline src/v2/layouts/MemberConfigSearch/MemberConfigSearch.tsx:1]: This should also reside under MemberDataGrid

File: src/v2/layouts/MemberConfigSearch/MemberConfigSearch.tsx [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+
+	const onChangeSearchText: InputChangeEventHandler = (event) => {
+		if (searchText !== event.target.value && isEmpty(event.target.value)) {
+			props.clearSearch();
+		}
+	};
+
+	const onChangeMultipleIdSearchCheckBox: InputChangeEventHandler = (event) => {
+		if (event.target.checked) {
+			setThresholdValue('0.0');
```
Comment (pepcus-mohit) [inline src/v2/layouts/MemberConfigSearch/MemberConfigSearch.tsx:47]: Are we updating this threshold value anywhere? If not then no need to create state for it.

File: src/v2/layouts/MemberRegistration/InvokeMemberRegistration.tsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import { useAPI } from '../../components/hooks';
```
Comment (pepcus-mohit) [inline src/v2/layouts/MemberRegistration/InvokeMemberRegistration.tsx:1]: I've deleted this file in the latest develop. Have you merged latest develop on your branch??

File: src/v2/utils/dataGridUtils.ts [hunk 5]
```diff
@@ -0,0 +1,69 @@
+import Fuse from 'fuse.js';
```
Comment (pepcus-mohit) [inline src/v2/utils/dataGridUtils.ts:1]: we can also move this file to MemberDataGrid directory

**Review by pepcus-mohit — CHANGES_REQUESTED** [review]

File: src/v2/utils/dataGridUtils.ts (hunk 5 above)
Comment (pepcus-priya) [reply src/v2/utils/dataGridUtils.ts:1]: this file already placed inside the utils directory so why are we moving this to MemberDataGrid? Instead can we create MemberDataGrid util directory inside the utils folder and put this file into the same directory?

**Review by pepcus-mohit — APPROVED** [review]

**Pull request merged**